function_calling/src/tools.py

```diff
@@ -214,8 +215,22 @@ def query_db(query):
     #         'top_k': 1,
     #         'top_p': 1.0,
     #         'do_sample': False,
+    #         'process_prompt': False,
     #     },
     # )
+    # Using SambaStudio CoE expert as model for generating the SQL Query
+    llm = SambaStudio(
```

Comment:
Can you move this to a config? If there are any other vars that need to be moved I'd suggest moving them there so this kit has a config.yaml to match the othere